src/apps/testapps/testCellsToLinkedMultiPolygon.c

```diff
@@ -72,6 +72,24 @@ SUITE(cellsToLinkedMultiPolygon) {
         H3_EXPORT(destroyLinkedMultiPolygon)(&polygon);
     }
 
+    TEST(pentagonChildren) {
+        // children of pentagon 0x80ebfffffffffff
+        H3Index kids[] = {0x81ea3ffffffffff, 0x81eabffffffffff,
+                          0x81eafffffffffff, 0x81eb3ffffffffff,
+                          0x81eb7ffffffffff, 0x81ebbffffffffff};
+        int numCells = ARRAY_SIZE(kids);
```

Comment:
does this happen for any set of res 1 children of a res 0 pentagon?

Reply:
At res 2, exactly two pentagons of the 12 do not work: `8009fffffffffff` and `80c3fffffffffff`